### ISSUE TITLE
planemo shed_upload --tar_only doesn't need --shed_target

### ISSUE THREAD

**Issue by peterjc** [galaxyproject/planemo#410]
e.g.

```
$ planemo --version
planemo, version 0.20.1.dev0
$ planemo shed_upload --tar_only  ~/repositories/pico_galaxy/tools/coverage_stats/
Usage: planemo shed_upload [OPTIONS] PROJECT

Error: default_shed_target set to None, must specify a value for --shed_target to run this command.
```

Since the construction of the tar-ball is offline, there should be no reason to require a target Tool Shed (although in general this is needed for the ``planemo shed_upload`` command).

Workaround, give a dummy ``--shed_target`` or ``-t`` option:

```
$ planemo shed_upload -t xxx --tar_only  ~/repositories/pico_galaxy/tools/coverage_stats/
cp '/tmp/tmpYuuev5' 'shed_upload.tar.gz'
```

**Comment by jmchilton** [comment]
Good point. The work around is to use ``shed_build`` I think, this should be fixed but if the fix isn't clean maybe it is time to deprecate ``--tar_only`` in lieu to ``shed_build``.

**Comment by peterjc** [comment]
Deprecation of ``planemo shed_upload --tar_only ...`` in favour of ``planemo shed_build ...`` seems fine, although it is quite recent added in v0.15.0 (2015-10-01).

**Comment by peterjc** [comment]
We can't yet replace ``planemo shed_upload --tar_only ...`` with ``planemo shed_build ...`` because the later doesn't work as expected, see #414.
